packages/bpk-component-page-indicator/README.md

```diff
@@ -0,0 +1,38 @@
+# bpk-component-page-indicator
+
+> Backpack example component.
+
+## Installation
+
+```sh
+npm install bpk-component-page-indicator --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkPageIndicator from 'bpk-component-code';
+
+export default () =>
+  <BpkPageIndicator
+    currentItem={0}
+    totalItems={7}
+    updateItemCallback={(newIndex) => {
+      console.log('Slide to new index:', newIndex);
+    }}
+  />;
+```
+
+## Props
```

Comment:
You missed the key required props and functions, otherwise how can we used it for displaying multiple swappable images?

Reply:
Have removed the swipeable library, thank you!